src/types.js

```diff
@@ -799,6 +799,9 @@ export type JSXAttribute = Node;
 export type JSXOpeningElement = Node;
 export type JSXClosingElement = Node;
 export type JSXElement = Node;
+export type JSXOpeningFragment = Node;
+export type JSXClosingFragment = Node;
+export type JSXFragment = Node;
```

Comment:
Ah, that's why there are so many `$FlowIgnore` in the JSX parser. None of these types are written 😆

(Even if you feel inclined to, don't do it on this PR, of course)